src/api/spec/support/coverage.rb

```diff
@@ -1,9 +1,14 @@
 # SimpleCov configuration
 SimpleCov.start 'rails' do
+  ENV['CODECOV_FLAG'] = ENV['TEST_SUITE']
   # NOTE: Keep filters in sync with test/test_helper.rb
   add_filter '/app/indices/'
   add_filter '/lib/templates/'
   add_filter '/lib/memory_debugger.rb'
   add_filter '/lib/memory_dumper.rb'
   merge_timeout 3600
 end
+
+# for tracking test coverage
+require 'codecov'
```

Comment:
This `require` shouldn't be at the beginning of the file?

Reply:
Let me check. So far I just picked the codecov related code that got dropped in ce192d25f2f6949265a2952db865f6502c05e428